src/OrbitQt/CallTreeViewItemModel.cpp

```diff
@@ -138,6 +156,16 @@ QVariant CallTreeViewItemModel::GetEditRoleData(const QModelIndex& index) const
       case kOfParent:
         return unwind_errors_item->GetPercentOfParent();
     }
+  } else if (unwind_error_type_item != nullptr) {
+    switch (index.column()) {
+      case kThreadOrFunction:
+        return QString::fromStdString(
+            orbit_client_data::CallstackTypeToString(unwind_error_type_item->error_type()));
```

Comment:
The edit role is used for sorting. This means that we sort alphabetically among siblings. I think this is fine, but if you want some other behaviour here's where to change it.

For example, as you can see the edit role when `unwind_errors_item != nullptr` is `QVariant();`. I guess I didn't want the "[unwind errors]" string to participate in sorting with the function names.

Reply:
I think for this case here, this is how we want it to be.